CDP4DalJsonSerializer/CDP4DalJsonSerializer.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net47;net471;net472;net48;netstandard2.0;netstandard2.1</TargetFrameworks>

```

Comment:
why do we need a new lib?

Reply:
This was introduced by Antoine to separate things to another library. Don't know the exact details. @antoineatstariongroup could you explain?

Reply:
ok